frontend/src/components/funding/FundingAgreementsTab.vue

```diff
@@ -59,12 +60,43 @@ export default {
     ...mapState(useAuthStore, ['userInfo']),
   },
 
-  created() {
+  async created() {
+    this.BASE_FUNDING = 'Base Funding'
+    this.loading = true
     this.format = format
     this.FUNDING_AGREEMENT_STATUS_CODES = FUNDING_AGREEMENT_STATUS_CODES
```

Comment:
If constants aren't accessed in the template, they don't need to be redefined in created().